registration/serializers.py

```diff
@@ -77,6 +77,7 @@ class FamilyDetailSerializer(serializers.HyperlinkedModelSerializer):
     children = StudentSerializer(many=True)
     guests = StudentSerializer(many=True)
     current_enrolment = SerializerMethodField()
+    enrolments = SerializerMethodField()
```

Comment:
we probably don't need this logic! instead, we can make enrolments reference the `EnrolmentSerializer` directly. 
this is very similar to how we get nested data about classes in the sessions serializer! ([see here](https://github.com/uwblueprint/project-read-backend/blob/fbeeea154c5a99b7d24b71034c6fae161ac20602/enrolments/serializers.py#L39))

you'll notice that we'll get some circular import errors when importing from the enrolment serializer - I think we can fix this by removing [the FamilySerializer import at the top of enrolments/serializers](https://github.com/uwblueprint/project-read-backend/blob/86cae9be98e47d86b15f96dfb1b173b936839d6a/enrolments/serializers.py#L4) and importing it only when it's called instead ([call 1](https://github.com/uwblueprint/project-read-backend/blob/86cae9be98e47d86b15f96dfb1b173b936839d6a/enrolments/serializers.py#L48), [call 2](https://github.com/uwblueprint/project-read-backend/blob/86cae9be98e47d86b15f96dfb1b173b936839d6a/enrolments/serializers.py#L73)) - will be very similar to [this](https://github.com/uwblueprint/project-read-backend/blob/86cae9be98e47d86b15f96dfb1b173b936839d6a/registration/serializers.py#L68)
```suggestion
    enrolments = EnrolmentSerializer(many=True)
```